### ISSUE TITLE
Improve the UI and color scheme of the single blog post

### ISSUE THREAD

**Issue by Alecaddd** [thundernest/thunderblog#30]
The single blog post page, especially when there's a lot of content, looks a bit blend and everything squishes together without much visual separation. The visual readability overall is not great.

We should improve this page by defining better colors, typography, and styling default blocks.

Some things that should be explored are:
- Better typography sizing
- Better line height and paragraph spacing
- Links should be colored and have a distinct hover effect
- Donation block should be custom styled

Something else?
@micahilbery, also this might need your magic touch.

**Comment by devmount** [comment]
I agree with all of the mentioned points. If we have some kind of style guide, I can happily implement it.

**Comment by micahilbery** [comment]
No, unfortunately I haven't gotten to take much of a look at this. However I imaging the blog visuals will be given more priority when the main site is taken care of in order to match it better. When that happens these typography things should also be addressed.
